client/src/components/pages/ConcertPage/ConcertPage.scss

```diff
@@ -14,7 +16,7 @@
 
 .header > div {
   color: #000;
-  font-family: Georgia;
+  font-family: $heading-2-font;
   font-weight: 700;
   letter-spacing: 0.8px;
   line-height: 73px;
```

Comment:
with the font-change, could you increase the `line-height` here? 
<img width="951" alt="screen shot 2018-11-27 at 4 19 09 pm" src="https://user-images.githubusercontent.com/22732325/49120485-b6f41880-f261-11e8-9529-015cef15d4e8.png">